openshift/release/create-release-branch.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Usage: create-release-branch.sh v0.4.1 release-0.4
+
+set -ex # Exit immediately on error.
+
+release=$1
+target=$2
+
+# Fetch the latest tags and checkout a new branch from the wanted tag.
+git fetch upstream -v --tags
+git checkout -b "$target" "$release"
+
+# Remove GH Action hooks from upstream
+rm -rf .github/workflows
+git commit -sm ":fire: remove unneeded workflows" .github/
+
+# Copy the openshift extra files from the OPENSHIFT/main branch.
+git fetch openshift main
+git checkout openshift/main -- .github/workflows openshift OWNERS OWNERS_ALIASES Makefile
```

Comment:
Need to add the https://github.com/openshift-knative/backstage-plugins/blob/release-v1.12/.github/workflows/release-plugins.yaml to main

Actually, all of the midstream workflows from v112 to main